<launch>
  <arg name="camera_action_service_name" default="camera_action"/>
  <arg name="waypoint_reached_topic_name" default="waypoint_reached"/>

  <arg name="output_directory" default="/tmp"/>
  <arg name="frame_id_pairs" default="$(find dji_camera)/config/frame_id_pairs.yml" />

  <node name="dji_camera" pkg="dji_camera" type="dji_camera.py" respawn="true" output="screen">

    <remap from="camera_action" to="$(arg camera_action_service_name)" />
    <remap from="waypoint_reached" to="$(arg waypoint_reached_topic_name)" />

    <param name="frame_id_pairs" type="str" value="$(arg frame_id_pairs)"/>
    <param name="output_directory" type="str" value="$(arg output_directory)"/>

  </node>
</launch>
